.gitignore

```diff
@@ -7,5 +7,8 @@ feature_engine/modules/
 feature_engine/__pycache__/
 feature_engine/features/__pycache__/
 feature_engine/ipython_notebooks/.ipynb_checkpoints/
-*.py[cod]
+team_process_map/
```

Comment:
@yashveersinghsohi wait, what happens when you add the directory itself to the .gitignore? or did you create a local folder inside `team_process_map/` that is also named `team_process_map/`?

Reply:
So I have a virtual python environment setup that by default takes the name of the project - `team_process_map`
I do believe that the `.gitignore` file would look for a `team_process_map` directory inside the `team_process_map` folder (and not find anything for anyone else other than my local system) hence we should be good.

Although this is confusing and I can clean this one up in a future PR.
Thanks for flagging.